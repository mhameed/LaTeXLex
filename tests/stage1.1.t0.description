author: mmh
date: 09/02/28
description: 
Pulling up dollar environment onto same line (except for inside verbatims).
