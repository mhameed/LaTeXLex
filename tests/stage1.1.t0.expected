Hi this is a test in r^n space.
Here comes a %comment.
%dollars split over 3 lines.
f(x) := x^2+2x+1 
\begin{verbatim}
%this is a comment.
$f(x) :=
x^2+2x+1
$
\end{verbatim}
